src/lib/updating-element.ts

```diff
@@ -653,12 +654,12 @@ export abstract class UpdatingElement extends HTMLElement {
    * `updateComplete` promise. This promise can be overridden and is therefore
    * not free to access.
    */
-  private _requestUpdate(name?: PropertyKey, oldValue?: unknown) {
+  private _requestUpdate(name?: PropertyKey, oldValue?: unknown, options?: PropertyDeclaration) {
     let shouldRequestUpdate = true;
     // If we have a property key, perform property update steps.
     if (name !== undefined) {
       const ctor = this.constructor as typeof UpdatingElement;
-      const options = ctor.getPropertyOptions(name);
+      options = options || ctor.getPropertyOptions(name);
```

Comment:
In a corner case, this is an observable change. It prevents the property options from being replaced without also overriding `_requestUpdate`.

Reply:
There is no API for mutating property options so you'd have to hold onto and mutate the object yourself. I don't think it's reasonable to assume that was ever supported.

Reply:
There's already a note on `getPropertyOptions` that says
> Note, this method should be considered "final" and not overridden. To customize the options for a given property, override `createProperty`.
So this seems safe to me.